terra/Makefile

```diff
@@ -0,0 +1,55 @@
+bridge_SOURCE=wormhole
+token_bridge_SOURCE=token_bridge
+nft_bridge_SOURCE=nft_bridge
+
+SOURCE_FILES=$(shell find . -name "*.rs") $(shell find . -name "*.lock") $(shell find . -name "*.toml") Dockerfile
+
+PACKAGES=$(shell find . -name "Cargo.toml" | grep -E 'packages|contracts' | cut -d/ -f3 | sed s/-/_/g)
+WASMS=$(patsubst %, artifacts/%.wasm, $(PACKAGES))
+
+.PHONY: artifacts
+## Build contracts.
+artifacts: artifacts/checksums.txt
+
+VALID_mainnet=1
+VALID_testnet=1
+VALID_devnet=1
+.PHONY: check-network
+check-network:
+ifndef VALID_$(NETWORK)
+	$(error Invalid or missing NETWORK. Please call with `$(MAKE) $(MAKECMDGOALS) NETWORK=[mainnet | testnet | devnet]`)
+endif
+
+$(WASMS) artifacts/checksums.txt: $(SOURCE_FILES)
+	docker build --target artifacts -o artifacts .
```

Comment:
Can we do a --no-cache on the build action?  I realize it will take more time per run, but I'm overly cautious about shipping cached artifacts.  Similarly, I'd also suggest having this step should call clean, for similar reasons.

Reply:
I think independent verification should eliminate any issues from this, and no-cache is too painful during development.